website/docs/r/nsxt_alb_settings.html.markdown

```diff
@@ -46,10 +68,20 @@ The following arguments are supported:
   [vcd_nsxt_edgegateway](/providers/vmware/vcd/latest/docs/data-sources/nsxt_edgegateway) data source
 * `is_active` - (Required) Boolean value `true` or `false` if ALB is enabled. **Note** Delete operation of this resource
   will set it to `false`
+* `supported_feature_set` - (Optional; *v3.7+*) Feature set of this Edge Gateway if ALB is enabled (`STANDARD` or `PREMIUM`)
+* `is_transparent_mode_enabled` - (Optional; *v3.9+*, *VCD 10.4.1+*) When enabled, it allows to
+  configure Preserve Client IP on a Virtual Service
 * `service_network_specification` - (Optional) Gateway CIDR format which will be used by Load Balancer service. All the
   load balancer service engines associated with the Service Engine Group will be attached to this network. The subnet
   prefix length must be 25. If nothing is set, the **default is 192.168.255.125/25**. This field cannot be updated
-* `supported_feature_set` - (Optional; *v3.7+*) Feature set of this Edge Gateway if ALB is enabled (`STANDARD` or `PREMIUM`)
+* `ipv6_service_network_specification` (Optional; *v3.9+*, *VCD 10.4.0+*) The IPv6 network
+  definition in CIDR format which will be used by Load Balancer service in the Edge Gateway. All the
+  load balancer service engines associated with the Service Engine Group will be attached to this
+  network. This field cannot be updated
+
+~> IPv4 service network will be used if both the `service_network_specification` and
+`ipv6_service_network_specification` properties are unset. If both are set, it will still be one
+service network with a dual IPv4 and IPv6 stack.
```

Comment:
And what if `service_network_specification` is unset, but `ipv6_service_network_specification` is set?

Reply:
Good question, which is not directly documented anywhere, but it uses only IPv6 then as per UI. I have put an explicit test step for that as well.

Setting only IPv6 in UI
<img width="569" alt="image" src="https://user-images.githubusercontent.com/15804230/220024761-62e9fa01-0941-4ccd-9d64-b82b00d5486e.png">

Results in such configuration as well

<img width="823" alt="image" src="https://user-images.githubusercontent.com/15804230/220024792-256e8ace-670c-4b17-ae59-ab60107348d1.png">